Backend/dev/mock-servers/server.py

```
@@ -940,24 +1060,68 @@ def _read_body(self):
+        if isinstance(exc, (ConnectionResetError, BrokenPipeError, ConnectionAbortedError)):
+            return
+        super().handle_error(request, client_address)
+
+
 def main():
     parser = argparse.ArgumentParser(description="Unified mock server for NammaYatri")
     parser.add_argument("--port", type=int, default=8080, help="Port (default: 8080)")
     args = parser.parse_args()
 
-    server = HTTPServer(("0.0.0.0", args.port), MockHandler)
+    server = _QuietThreadingHTTPServer(("0.0.0.0", args.port), MockHandler)
```

Ruff (0.15.15)

[error] Possible binding to all interfaces

(S104)